File: Crtm05/Parameters.cs
using System;

namespace Crtm05
{
    public class Parameters : IParameters
    {
        const double latitudeOrigin = 10;
        const double longitudeOrigin = -84;
        const double la = 1105855;
        const double fe = 500000;
        const double rho = Math.PI / 180;

        public double LatitudeOrigin => latitudeOrigin;
        public double LongitudeOrigin => longitudeOrigin;
        public double LA => la;
        public double FE => fe;
        public double Rho => rho;
    }
}

File: typescript/Crtm05/parameters.ts
import { IParameters } from './iparameters';

const _latitudeOrigin = 10;
const _longitudeOrigin = -84;
const _la = 1105854.83321889;
const _fe = 500000;
const _rho: number = Math.PI / 180;

export class Parameters implements IParameters {
  latitudeOrigin: number = _latitudeOrigin;
  longitudeOrigin: number = _longitudeOrigin;
  la: number = _la;
  fe: number = _fe;
  rho: number = _rho;
}
